src/okteto.ts

```diff
@@ -1,3 +1,33 @@
+const contexts: Fig.Generator = {
+  script: "okteto context list",
```

Comment:
This is how we can add cache to a generator
```suggestion
  script: "okteto context list",
  cache: {
    ttl: 1000 * 60 * 60 * 24 * 2 // 2 days
  },
```

Reply:
Thanks @fedeci! If this `ttl` is 2 days, does this mean that even if new `contexts` are available, they won't be shown until the `ttl` expires, and we request for the `contexts` again?

Reply:
That's what we do!

Reply:
@rberrelleza do we want this? This would mean new `contexts` don't show up immediately. We could reduce the `ttl` to an hour to make it more convenient, but I could still see this being problematic in cases where a newly created `context` won't load up in the suggestions immediately.

Reply:
@RinkiyaKeDad I think that we should have a cache. 30 minutes sounds reasonable as a starting point. Once cache invalidation is available, we can rethink this approach.